packages/react/src/components/Storage/FileUploader/hooks/useFileUploader/useFileUploader.ts

```diff
@@ -4,17 +4,12 @@ import { checkMaxSize, returnAcceptedFiles } from '@aws-amplify/ui';
 import { Files, FileState, FileStatuses } from '../../types';
 import { UseFileUploader } from './types';
 
-export default function useFileUploader({
-  maxSize,
+export const useFileUploader: UseFileUploader = ({
```

Comment:
Prefer named functions for larger utils, and it better aligns with existing conventions as noted in the earlier comment about usage of PascalCase and hooks names

Reply:
updated